ci/stage-test-examples.jenkinsfile

```diff
@@ -0,0 +1,117 @@
+stage('test-examples') {                        
+    if (env.SGX == "1") {
+        env.MODE = "sgx"
+    }
+    else {
+        env.MODE = "direct"
+    }
+
+    try {
+        timeout(time: 5, unit: 'MINUTES') {
+            sh '''
+                cd CI-Examples/rust_helloworld
+                export PATH=/home/intel/rust_binaries/bin:$PATH
+                make run-release-${MODE} 2>&1 | tee OUTPUT.txt
+            '''
+        }
+    } catch (Exception e){
+        env.build_ok = false
+        sh 'echo "Rust example Test Failed"'
+    }
+
+    try {
+        timeout(time: 5, unit: 'MINUTES') {
+            sh '''
+                cd CI-Examples/openjdk
+                make ${MAKEOPTS} ${ARCH_LIB_OPT} all
+                gramine-${MODE} java MultiThreadMain > OUTPUT
+            '''
+        }
+    } catch (Exception e) {
+        env.build_ok = false
+        sh 'echo "Openjdk Example Test Failed"'
+    }
+
+    /*
+    try {
+        timeout(time: 15, unit: 'MINUTES') {
+            sh '''
+                cd CI-Examples/tensorflow-lite
```

Comment:
didn't you say tf is not working?

Reply:
It has been commented and result verification is marked as skip in workload_parser. I didn't want to lose the changes for this workload because we will fix this issue sooner